### DISCUSSION TITLE
Ständiger Crash und alles gelöscht.

### DISCUSSION THREAD

**Discussion by delimedia** [jomjol/AI-on-the-edge-device#2678] [General] [answered]
### Issue

Ich habe mit extra zwei  ESP32-CAM geholt. 
Diese hier: https://www.amazon.de/dp/B093GSCBWJ?psc=1&ref=ppx_yo2ov_dt_b_product_details

Einmal für den Gaszähler und einmal für den Wasserzähler.
Mittlerweile ist meine Zündschnur echt kurz geworden... 
Ich habe alles nach Anleitung gemacht  .... mehrfach...
Der längste Versuch hat jetzt knapp 20 Stunden funktioniert. 
Also entweder haben beide ESP Module einen an der Waffel oder die SD Karten sind nicht ihr Geld wert.
Aber was ist passiert.
Wenn die Einrichtung funktioniert hat war es schon viel und ja nicht zu schnell die Seiten wechseln sonst war alles von der Programmierung hin. 
Meist nach dem ersten Reboot ging einfach nichts mehr... Fehler Blinkcode 8x2
Es gab auch Momente da lief es mehrere Stunden und danach war dann alles wieder hin.
Also SD-Karte raus rein in den PC und siehe da es sieht so aus:
https://postimg.cc/Lq950tpb

Die Logs sind so gut wie leer und sehen so aus:

> [0d00h00m00s] 1970-01-01T00:00:00	<INF>	[MAIN] =================================================
> [0d00h00m00s] 1970-01-01T00:00:01	<INF>	[MAIN] ==================== Start ======================
> [0d00h00m00s] 1970-01-01T00:00:01	<INF>	[MAIN] =================================================
> [0d00h00m00s] 1970-01-01T00:00:01	<INF>	[MAIN] PSRAM size: 8388608 byte (8MB / 64MBit)
> [0d00h00m00s] 1970-01-01T00:00:01	<INF>	[MAIN] Total heap: 4376891 byte
> [0d00h00m05s] 1970-01-01T00:00:05	<INF>	[MAIN] Camera info: PID: 0x26, VER: 0x42, MIDL: 0x7f, MIDH: 0xa2
> [0d00h00m05s] 1970-01-01T00:00:05	<INF>	[SDCARD] Basic R/W check started...
> [0d00h00m05s] 1970-01-01T00:00:05	<INF>	[SDCARD] Basic R/W check successful
> [0d00h00m05s] 1970-01-01T00:00:05	<ERR>	[MAIN] Config file seems to be missing!
> [0d00h00m05s] 1970-01-01T00:00:05	<WRN>	[SNTP] No ConfigFile defined - exit setupTime()!
> [0d00h00m05s] 1970-01-01T00:00:05	<WRN>	[MAIN] No ConfigFile defined - exit setCpuFrequency()!
> [0d00h00m00s] 1970-01-01T00:00:00	<INF>	[MAIN] =================================================
> [0d00h00m00s] 1970-01-01T00:00:00	<INF>	[MAIN] ==================== Start ======================
> [0d00h00m00s] 1970-01-01T00:00:00	<INF>	[MAIN] =================================================
> [0d00h00m00s] 1970-01-01T00:00:00	<INF>	[MAIN] PSRAM size: 8388608 byte (8MB / 64MBit)
> [0d00h00m00s] 1970-01-01T00:00:00	<INF>	[MAIN] Total heap: 4376891 byte
> [0d00h00m04s] 1970-01-01T00:00:04	<INF>	[MAIN] Camera info: PID: 0x26, VER: 0x42, MIDL: 0x7f, MIDH: 0xa2
> [0d00h00m04s] 1970-01-01T00:00:04	<INF>	[SDCARD] Basic R/W check started...
> [0d00h00m04s] 1970-01-01T00:00:04	<INF>	[SDCARD] Basic R/W check successful
> [0d00h00m04s] 1970-01-01T00:00:04	<ERR>	[MAIN] Config file seems to be missing!
> [0d00h00m04s] 1970-01-01T00:00:04	<WRN>	[SNTP] No ConfigFile defined - exit setupTime()!
> [0d00h00m04s] 1970-01-01T00:00:04	<WRN>	[MAIN] No ConfigFile defined - exit setCpuFrequency()!

Hat da jemand eine Idee?
Wenn ich das Ding gefühlt jeden Tag neu bespielen muss. Kann ich die Zähler manuell ablesen.

**Comment by friedpa** [comment, marked as answer]
Entweder sind die SD-Karten nicht zu gebrauchen, oder aber die ESP können die SD-karten nicht lesen. Ich verwende nur mehr SanDisk Extreme Produkte und bei den ESPs verwende ich den deutschen Händler AZ-Delivery Vertriebs GmbH (az-delivery.com) = immer gute Ware. die genau das hält für was man bezahlt. Außerdem ist das Service, wenn etwas passieren sollte, sehr gut.

**Comment by delimedia** [comment]
Ok dann werde ich es damit mal probieren hatte jetzt die Intenso microSDHC 8GB Class 10  genommen weil die bei einigen Anleitungen empfohlen wurden.  Muss gesetehen war noch nie von intenso überzeugt :)

**Comment by friedpa** [comment]
Mit dieser, von mir beschriebenen, HW-Variante funktonieren mein Wasser- und mein Gaszähler seit mehr als zwei Jahren ohne einen einzigen Ausfall.
